_posts/2018-06-14-fio-tutorial.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Running FIO with pmem engines
+author: lukaszstolarczuk
+layout: post
+identifier: fio_tutorial
+---
+
+When we want to check performance of our library, either see if there was any regression or if our tweaks did the good job, we execute benchmarks. One of them is FIO. It helps us simulate synthethic traffic of reads and writes to pmem device. In this blog post we'll try to introduce this tool and explain how we use it.
```

Comment:
"when we, the PMDK team, want to"
"either to see"
"we run benchmarks"
"to a pmem device"
"I will introduce"
"how we commonly use it"